packages/salesforcedx-vscode-core/README.md

```diff
@@ -18,7 +18,7 @@ To run a command from Salesforce Extensions for VS Code, press Cmd+Shift+P (macO
 When a command finishes running (due to success, failure, or cancellation), a notification displays at the top of the window.  
 ![Notification that source was successfully pushed to a scratch org](https://raw.githubusercontent.com/forcedotcom/salesforcedx-vscode/develop/packages/salesforcedx-vscode-core/images/command_success_notification.png)
 
-To see the output of the commands that you run, select **View** > **Output**, and then select **Salesforce DX CLI** from the dropdown menu. Alternatively, click **Show** in the completion notification.  
+To see the output of the commands that you run, select **View** > **Output**, and then select **Salesforce CLI** from the dropdown menu. Alternatively, click **Show** in the completion notification.  
```

Comment:
@ruthemmanuelle - Just FYI. Any comments here?

Reply:
Thanks for fixing this, Gunnar.